examples/src/main/scala/org/apache/spark/examples/sql/SparkSQLExample.scala

```diff
@@ -60,7 +60,8 @@ object SparkSQLExample {
 
   private def runBasicDataFrameExample(spark: SparkSession): Unit = {
     // $example on:create_df$
-    val df = spark.read.json("examples/src/main/resources/people.json")
+    val df = spark.read.json(
+      "file:///" + System.getProperty("user.dir") + "examples/src/main/resources/people.json")
```

Comment:
@HyukjinKwon Thanks for your comment! Without HDFS configuration(including HADOOP_HOME), it result in failure.

```
./bin/run-example sql.SparkSQLExample
...
16/11/10 22:48:07 WARN datasources.DataSource: Error while looking for metadata directory.
Exception in thread "main" java.net.ConnectException: Call From ozamac-2.local/10.0.1.20 to localhost:9000 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
```

After applying the patch, it works correctly.